### PR TITLE
RGBDbugfix

### PR DESCRIPTION
**Pull request by aerydna**
-
-camInfo params for ros interoperation
-typo correction
-yarp::os::Node add "/" at the beginning of the name if not present.

### CODE REVIEW

**Comment by drdanz** [comment]
@aerydna please rebase the branch over the latest `master` (if it is a bugfix) or over the latest `devel` if it contains new features, there are several "Merge branch" commits and the "Prepare for next feature release (2.3.70)" commit that should not be there

**Comment by aerydna** [comment]
ok Sorry

File: src/libYARP_OS/src/Node.cpp [hunk 1]
```diff
@@ -549,13 +549,22 @@ Node::Node() :
 Node::Node(const ConstString& name) :
         mPriv(new Helper)
 {
+    ConstString _name;
+    if(name.at(0) != '/')
+    {
+        _name == "/"+name;
```
Comment (aerydna) [inline src/libYARP_OS/src/Node.cpp]: you're right, small oversight..

Comment (barbalberto) [inline src/libYARP_OS/src/Node.cpp]: Ok as a bugfux.
As a personal taste, I don't like so much when the the code changes user params on its will, and I think a constructor should never attempt operation which may fail, like opening port or check config params.

I propose to remove this option and substitute with (if not yet existing) an empty constructor and a init/config method which accepts the node name. This way the method can clearly fail and a better error handling can be achieved.

Can we chat a bit about this in the next IFI day?

File: src/libYARP_OS/src/Node.cpp [hunk 2]
```diff
@@ -549,13 +549,22 @@ Node::Node() :
...
 {
+    ConstString _name;
+    if(name.at(0) != '/')
+    {
+        _name = "/"+name;
+    }
+    else
+    {
+        _name = name;
+    }
```
Comment (drdanz) [inline src/libYARP_OS/src/Node.cpp]: Please, make a separate PR for this change, even if it is very small it is a change of behaviour in YARP_OS, and I don't want this to be hidden in a "RGBDbugfix" pull request...

**Comment by drdanz** [comment]
Merged, thanks.
